modules/vertx-swagger-codegen/src/main/resources/javaVertXServer/apiRXVerticle.mustache

```diff
@@ -0,0 +1,85 @@
+package {{package}};
+
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.json.Json;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+
+
+{{#imports}}import {{import}};
+{{/imports}}
+
+import java.util.List;
+import java.util.Map;
+
+public class {{classname}}Verticle extends AbstractVerticle {
+    final static Logger LOGGER = LoggerFactory.getLogger({{classname}}Verticle.class); 
+    
+    {{#operations}}{{#operation}}{{#vendorExtensions}}final static String {{x-serviceId-varName}} = "{{x-serviceId}}";
+    {{/vendorExtensions}}{{/operation}}{{/operations}}
+    //TODO : create Implementation
+    {{classname}} service = new {{classname}}Impl();
+
+    @Override
+    public void start() throws Exception {
+        {{#operations}}{{#operation}}
+        //Consumer for {{#vendorExtensions}}{{x-serviceId}}{{/vendorExtensions}}
+        vertx.eventBus().<JsonObject> consumer({{#vendorExtensions}}{{x-serviceId-varName}}{{/vendorExtensions}}).handler(message -> {
+            try {
+                {{#hasParams}}
+                    {{#allParams}}
+                        {{#isListContainer}}
+                {{{dataType}}} {{paramName}} = Json.mapper.readValue(message.body().getJsonArray("{{baseName}}").encode(), Json.mapper.getTypeFactory().constructCollectionType(List.class, {{{baseType}}}.class));
+                        {{/isListContainer}}
+                        {{^isListContainer}}
+                            {{#isPrimitiveType}}
+                                {{#isString}}
+                {{{dataType}}} {{paramName}} = message.body().getString("{{baseName}}");
+                                {{/isString}}
+                                {{^isString}}
+                {{{dataType}}} {{paramName}} = Json.mapper.readValue(message.body().getString("{{baseName}}"), {{{dataType}}}.class);
+                                {{/isString}}
+                            {{/isPrimitiveType}}
+                            {{^isPrimitiveType}}
+                {{{dataType}}} {{paramName}} = Json.mapper.readValue(message.body().getJsonObject("{{baseName}}").encode(), {{{dataType}}}.class);
+                            {{/isPrimitiveType}}
+                        {{/isListContainer}}
+                    {{/allParams}}
+                {{/hasParams}}
+                service.{{operationId}}({{#hasParams}}{{#allParams}}{{paramName}}{{#hasMore}}, {{/hasMore}}{{/allParams}}{{/hasParams}}).subscribe(
+                    onSuccess -> {
```

Comment:
I'd rename "onSuccess" to "result" (or something similar), as it holds the result object.
Same for "onError", which is better named "error" IMHO.